portfolio/src/main/webapp/script.js

```diff
@@ -54,3 +54,17 @@ function updateButtonsState(visibleImgId) {
     nextButton.disabled = false;
   }
 }
+
+function toggleTheme() {
+  var toggler = document.getElementById('theme-toggler');
+  var body = document.getElementsByTagName('body')[0];
+  if (toggler.checked) {
+    body.classList.remove('light-theme');
+    body.classList.add('dark-theme');
+  } else {
+    body.classList.remove('dark-theme');
+    body.classList.add('light-theme');
+  }
+}
+
+function scrollUp() { window.scrollTo({top : 0, behavior : "smooth"}); }
```

Comment:
Usually this should have been handled by linter I guess, but it's visually more clear to separate function body to new lines, even it's a one-line function.

Reply:
It was actually edited by clang linter, which is recommended by Google stylesheet as the best for JavaScript. Probably I need to set it up a bit